### ISSUE TITLE
Bug Report: News page fail to open (404)

### ISSUE THREAD

**Issue by JiaxiangBU** [JohnCoene/g2r#2]
https://g2r.dev/news/news/index.html

**Comment by hadley** [comment]
The whole site is down for me.

**Comment by JohnCoene** [comment]
@JiaxiangBU erroneous link in navbar has been removed, thanks for reporting.
@hadley sorry, perhaps the latest commit fixed things for you?

**Comment by hadley** [comment]
Nope, it still just hangs when I attempt to connect.

**Comment by JohnCoene** [comment]
Oh, I'm unsure what is wrong, it's pkgdown via netlify, perhaps you can access netlify's auto-generated URL [gallant-boyd-300e1f.netlify.com](https://gallant-boyd-300e1f.netlify.com/). I really hope this works because I have no idea why you cannot access it :(

**Comment by hadley** [comment]
Yeah, that works, so it must be something related to DNS resolution. But I don't know how to diagnose that, or why it would be different for me.
